docs/cloud-native-security/kspm-faq.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[kspm-faq]]
+== KSPM FAQ
+Frequently asked questions related to the Kubernetes Security Posture Management (KSPM) integration.
+
+[discrete]
+== General KSPM FAQ
+
+*Can I evaluate the security posture of my Amazon EKS clusters?*
+Yes. KSPM currently supports the security posture evaluation of Amazon EKS and unmanaged Kubernetes clusters.
+
+*How often is my cluster’s security posture evaluated?*
+Clusters are evaluated upon deployment of a KSPM integration, and every four hours after that.
```

Comment:
+1 on combining these sections per @nastasha-solomon. For a relatively short page, it probably doesn't need to be so heavily segmented and granular.

Also a possible edit for tone:
```suggestion
Clusters are evaluated when you deploy a KSPM integration, and every four hours after that.
```